ts/Stock/Navigator/NavigatorDefaults.ts

```diff
@@ -451,9 +451,20 @@ const NavigatorDefaults: NavigatorOptions = {
     xAxis: {
         /**
          * Additional range on the right side of the xAxis. Works similar to
-         * xAxis.maxPadding, but value is set in milliseconds.
+         * `xAxis.maxPadding`, but value is set in milliseconds, percentage or pixels.
+         *
+         * If it's a number, it is interpreted as milliseconds.
+         *
+         * If it's a percentage string,
+         * it is interpreted as percentages of axis length.
+         * An overscroll of 50% will make a 100px axis 50px longer.
+         *
+         * If it's a pixel string, it is interpreted as fixed pixel value,
+         * but limited to 90% of axis width.
+         *
```

Comment:
This doclet looks like the same as the above, so the same corrections would apply here.